v3.1/Dockerfile

```diff
@@ -15,8 +15,10 @@ ENV APACHE_RUN_USER=www-data \
     APACHE_METRICSLOG='/dev/null combined' \
     APACHE_METRICS_DENY_FROM='All' \
     APACHE_METRICS_ALLOW_FROM='127.0.0.0/255.0.0.0 ::1/128' \
-    MODSEC_AUDIT_LOG=/var/log/modsecurity/audit.log \
-    MODSEC_AUDIT_STORAGE=/var/log/modsecurity/audit/ \
+    MODSEC_AUDIT_LOG=/dev/stdout \
+    MODSEC_AUDIT_LOG_TYPE=Serial \
+    MODSEC_AUDIT_LOG_FORMAT=JSON \
+    MODSEC_AUDIT_STORAGE= \
```

Comment:
Unfortunately the `mkdir` command in the `entrypoint.sh` script fails with an empty `MODSEC_AUDIT_STORAGE`:

```
mkdir: cannot create directory '': No such file or directory
```

I'd propose to leave the default of `/var/log/modsecurity/audit/`

Reply:
Added the default again 👍